ls1intum/Artemis
# `General`: Make markdown editor buttons more compact

### Checklist
#### General


#### Client

### Motivation and Context


The height of the buttons above the markdown editor for writing messages/posts are unnecessarily high.

### Description

I removed some margin and padding to reduce the height of the buttons.

### Steps for Testing

Prerequisites:
- None

1. Log in to Artemis as any user
2. Check the markdown editor everywhere where you can write messages/posts

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage






Same as develop
### Screenshots


before:
<img width="758" alt="before" src="https://github.com/ls1intum/Artemis/assets/44754405/5c469c2b-4f63-4977-b039-8f22e33d5389">

after:
<img width="756" alt="after" src="https://github.com/ls1intum/Artemis/assets/44754405/38f60773-d17c-4264-aaf5-8b807f665329">
